aws/resource_aws_cloudfront_key_group.go

```diff
@@ -0,0 +1,146 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudfront"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAwsCloudFrontKeyGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsCloudFrontKeyGroupCreate,
+		Read:   resourceAwsCloudFrontKeyGroupRead,
+		Update: resourceAwsCloudFrontKeyGroupUpdate,
+		Delete: resourceAwsCloudFrontKeyGroupDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"comment": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"etag": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
A computed `id` attribute is implicitly handled by the Terraform Plugin SDK, which is why you do not see it typically implemented in resource schemas. 👍